pipeline/ingest/ephys.py

```diff
@@ -151,10 +107,12 @@ def _load(self, data, probe, npx_meta, rigpath):
         log.info('-- Start insertions for probe: {} - Clustering method: {} - Label: {}'.format(probe, method, clustering_label))
 
         assert len(trial_start) == len(trial_go)
-        
+
         # create probe insertion records
+        if ininto_archive:
+            prb_insert_exists = True
         try:
-            insertion_key, e_config_key = _gen_probe_insert(sinfo, probe, npx_meta)
+            insertion_key, e_config_key = _gen_probe_insert(sinfo, probe, npx_meta, prb_insert_exists=prb_insert_exists)
```

Comment:
if I'm not mistaken, ininto_archive and prb_insert_exists don't need to be separate within the scope of this function - e.g. could:
`_gen_probe_insert(sinfo, probe, npx_meta, prb_insert_exists=ininto_archive)`
Did cross-search usage within function scope but still might be overlooking something here..